feature-libs/product-configurator/rulebased/components/attribute/header/configurator-attribute-header.component.html

```diff
@@ -3,22 +3,32 @@
   {{ 'configurator.attribute.notVisibleAttributeMsg' | cxTranslate }}
 </div>
 
-<label
-  id="{{ createAttributeUiKey('label', attribute.name) }}"
-  [class.cx-required-error]="showRequiredMessageForDomainAttribute$ | async"
-  [attr.aria-label]="
-    !attribute.required
-      ? ('configurator.a11y.attribute'
-        | cxTranslate: { attribute: attribute.label })
-      : ('configurator.a11y.requiredAttribute'
-        | cxTranslate: { param: attribute.label })
-  "
-  ><span
-    [class.cx-required-icon]="attribute.required"
-    [attr.aria-describedby]="createAttributeUiKey('label', attribute.name)"
-    >{{ getLabel(expMode, attribute.label, attribute.name) }}</span
-  ></label
->
+<div class="cx-header-label-row">
+  <label
+    id="{{ createAttributeUiKey('label', attribute.name) }}"
+    [class.cx-required-error]="showRequiredMessageForDomainAttribute$ | async"
+    [attr.aria-label]="
+      !attribute.required
+        ? ('configurator.a11y.attribute'
+          | cxTranslate: { attribute: attribute.label })
+        : ('configurator.a11y.requiredAttribute'
+          | cxTranslate: { param: attribute.label })
+    "
+    ><span
+      [class.cx-required-icon]="attribute.required"
+      [attr.aria-describedby]="createAttributeUiKey('label', attribute.name)"
+      >{{ getLabel(expMode, attribute.label, attribute.name) }}</span
+    ></label
+  >
+
+  <cx-configurator-show-options
+    *ngIf="
+      attribute.domainOnDemand && attribute.uiType !== 'dropdown_lazy_load'
```

Comment:
Please, correct me but I think this part `attribute.uiType !== 'dropdown_lazy_load`' should be removed.
As far as I understand **_Show Options_** should always be displayed underneath the attribute name